interpreter/exec/eval.ml

```diff
@@ -325,22 +325,30 @@ let rec step (c : config) : config =
       | Resume xls, Ref (NullRef _) :: vs ->
         vs, [Trapping "null continuation reference" @@ e.at]
 
-      | Resume xls, Ref (ContRef (n, ctxt)) :: vs ->
+      | Resume xls, Ref (ContRef {contents = None}) :: vs ->
+        vs, [Trapping "continuation resumed twice" @@ e.at]
```

Comment:
Just a thought: in the future, we may want to make "multiple resume" and "null continuation" traps catchable exceptions.

Reply:
No, I don't think so. These are essentially symptoms of codegen bugs in the producer, so a fatal failure ought to be adequate, just as for out-of-bounds memory access. If the producer really legally expects a null somewhere, then it should perform an explicit check.